examples/example-react-native-app/Gemfile

```diff
@@ -3,4 +3,7 @@ source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby '2.7.5'
 
-gem 'cocoapods', '~> 1.11', '>= 1.11.2'
+# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
```

Comment:
😢

Reply:
yeah not that the iOS stuff here matters but oh well